### PR TITLE
Add instructions for PERSIST_DIR to grist-local-testing README

### PR DESCRIPTION
**Pull request by dsagal**
## Context

grist-local-testing README has a how-to-use section, which didn't work.

## Proposed solution

Explain in README the extra step needed to make the minimal example work.

## Has this been tested?

I tested the new instruction. There are no code changes here.

- [ ] 👍 yes, I added tests to the test suite
- [ ] 💭 no, because this PR is a draft and still needs work
- [x] 🙅 no, because this is not relevant here
- [ ] 🙋 no, because I need help

### CODE REVIEW

File: docker-compose-examples/grist-local-testing/README.md [hunk 1]
```diff
@@ -9,4 +9,12 @@ See https://support.getgrist.com/self-managed for more information.
 
 ## How to run this example
 
-This example can be run with `docker compose up`.
\ No newline at end of file
+Before running, create a directory where Grist will store its documents and settings:
```
Comment (fflorent) [inline docker-compose-examples/grist-local-testing/README.md:12]: Nitpicking:
```suggestion
Before running the first time, create a directory where Grist will store its documents and settings:
```

Comment (fflorent) [reply docker-compose-examples/grist-local-testing/README.md:12]: As an alternative, may we create the directory ourselves?
```bash
$ touch docker-compose-examples/grist-local-testing/persist/.gitkeep
$ git add -f docker-compose-examples/grist-local-testing/persist/.gitkeep # force the file to be indexed, ignoring .gitignore
```

And also this change to make `PERSIST_DIR` equal to `./persist` in `docker-compose.yml` by default:
```diff
diff --git a/docker-compose-examples/grist-local-testing/docker-compose.yml b/docker-compose-examples/grist-local-testing/docker-compose.yml
index 028d4e0f..5ccf7399 100644
--- a/docker-compose-examples/grist-local-testing/docker-compose.yml
+++ b/docker-compose-examples/grist-local-testing/docker-compose.yml
@@ -3,6 +3,6 @@ services:
     image: gristlabs/grist:latest
     volumes:
       # Where to store persistent data, such as documents.
-      - ${PERSIST_DIR}/grist:/persist
+      - ${PERSIST_DIR:-./persist}/grist:/persist
     ports:
       - 8484:8484
```

What do you think?

**Review by fflorent — APPROVED** [review]
Looks good to me. I made few suggestions that can be ignored.

**Comment by dsagal** [comment]
Thanks for the suggestsions, @fflorent ! I used both, and also added some hopefully-helpful instructions to the README (you can see them rendered [here](https://github.com/gristlabs/grist-core/blob/update-docker-compose-readme/docker-compose-examples/grist-local-testing/README.md)).

@Spoffy , what do you think?

**Review by Spoffy — APPROVED** [review]
Looks good to me, a nice usability improvement!

**Pull request merged**